code/modules/admin/verbs/adminpm.dm

```diff
@@ -95,10 +131,16 @@
 
 	if(C.holder)
 		if(holder)	//both are admins
+			for(var/datum/adminticket/T in admintickets)
+				if(T.permckey == C.ckey && T.resolved == "No")
+					T.logs += "[src] TO [C]: [msg] "
```

Comment:
for logging purposes,perhaps make pms from an admin to an user of a certain colour,for example red, while viceversa in another color,like..um...blue?